code/datums/uplink/ammunition.dm

```diff
@@ -189,6 +189,31 @@
 	item_cost = 2
 	path = /obj/item/ammo_magazine/m12/pellet
 
+////Grenade Shells////
+
+/datum/uplink_item/item/ammo/grenade_baton
+	name = "6xbaton rounds"
+	desc = "A box containing 6 baton rounds. Designed to be fired from a grenade launcher."
+	item_cost = 4
+	path = /obj/item/weapon/storage/box/baton_rounds
+/datum/uplink_item/item/ammo/grenade_emp
+	name = "6xemp grenade shells"
```

Comment:
Change to EMP grenade shells x6, same for the other uplink items you're adding

Reply:
I named them in the same way actual grenades inside the upload are named, want me to change those names too while I am at it?